### PR TITLE
Add tests for server/renderers.py

### PR DESCRIPTION
**Pull request by ryanlerch**
Adds a new test for server/renderers.py, testing the JPEG
renderer.

### CODE REVIEW

File: bodhi/tests/server/test_renderers.py [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+            'captcha.background_color': '#ffffff',
+            'captcha.padding': '5',
+            'captcha.ttl': '300',
+        })
+        app = TestApp(main({}, session=self.db, **settings))
+
+        res = app.get('/updates/FEDORA-2017-a3bbe1a8f2', headers=dict(accept='text/html'))
+        captcha_url = re.search(r'"http://localhost(/captcha/[^"]*)"', str(res)).groups()[0]
+        resp = app.get(captcha_url)
+        self.assertIn('image/jpeg', resp.headers['Content-Type'])
```
Comment (bowlofeggs) [inline bodhi/tests/server/test_renderers.py]: We could probably use Pillow to assert that the image is a valid JPEG. Perhaps we could use [```pillow```](https://pillow.readthedocs.io/en/4.1.x/) to load the image and assert that it's is 300x80 pixels?

I'd say this is optional if you don't feel like doing it right now - we can always do it later.

File: bodhi/tests/server/test_renderers.py [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+            'captcha.font_color': '#000000',
+            'captcha.background_color': '#ffffff',
+            'captcha.padding': '5',
+            'captcha.ttl': '300',
+        })
+        app = TestApp(main({}, session=self.db, **settings))
+
+        res = app.get('/updates/FEDORA-2017-a3bbe1a8f2', headers=dict(accept='text/html'))
+        captcha_url = re.search(r'"http://localhost(/captcha/[^"]*)"', str(res)).groups()[0]
+        resp = app.get(captcha_url)
```
Comment (bowlofeggs) [inline bodhi/tests/server/test_renderers.py]: On both calls to ```app.get()```, you can put a ```status=200``` to also assert that there aren't error codes.

**Review by bowlofeggs — APPROVED** [review]
Let's put the status=200 calls on there, and I'll leave it to you if you want to try to load the jpeg with pillow or not.

**Comment by ryanlerch** [comment]
Thanks for the review, @bowlofeggs !

I have added the status checks, and also checking the imagesize using PIL.

merging now.

cheers,
ryanlerch

**Pull request merged**